src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -599,7 +658,9 @@ private void populateSummaryStatisticsFromResponses(
             FeedbackSessionResultsBundle bundle) {
         List<String> hiddenRecipients = new ArrayList<>(); // List of recipients to hide
         FeedbackParticipantType type = question.recipientType;
+
         for (FeedbackResponseAttributes response : responses) {
+            if (!bundle.visibilityTable.containsKey(response.getId())) continue;
```

Comment:
I think this part is handled by another branch.

Reply:
Thanks for the comments! Should be fixed now.